docs/azure/sdk/logging.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Logging with the Azure SDK for .NET
+description: Learn how to enable logging with the Azure SDK for .NET client libraries
+ms.date: 03/20/2020
+ms.custom: azure-sdk-dotnet
+ms.author: casoper
+author: camsoper
+---
+
+# Logging with the Azure SDK for .NET
+
+The [Azure SDK](https://azure.microsoft.com/downloads/) for .NET client libraries includes the ability to log client library operations. This allows you to monitor I/O requests and responses that client libraries are making to Azure services. Typically, the logs are used to debug or diagnose communication issues. This article describes three approaches to enable logging with the Azure SDK for .NET:
+
+- Log to the console window
+- Log to .NET diagnostics traces
+- Configure custom logging
+
+> [!NOTE]
+> This article applies to client libraries that use the most recent versions of the Azure SDK for .NET. To see if a library is supported, refer to the list of [Azure SDK latest releases](https://azure.github.io/azure-sdk/releases/latest/js.html). If your application is using an older version of the Azure SDK client libraries, refer to specific instructions in the applicable service documentation.
```

Comment:
I really like the phrasing of this. I think I'd call it out with a [!IMPORTANT] instead though.